app/controllers/katello/api/v2/upstream_subscriptions_controller.rb

```diff
@@ -18,6 +18,8 @@ class Api::V2::UpstreamSubscriptionsController < Api::V2::ApiController
       N_("List available subscriptions from Red Hat Subscription Management")
     param :organization_id, :number, :desc => N_("Organization ID"), :required => true
     param_group :cp_search
+    param :pool_ids, Array, desc: N_("List of pool ids to fetch")
+    param :quantities_only, :bool, desc: N_("Only returns id and quantity fields")
```

Comment:
If I send quantities_only=false it behaves as true - I think you need to do the ::Foreman::Cast.to_bool on that param

Reply:
I also experienced this issue.